countNumberOfLowercase.js

```diff
@@ -0,0 +1,13 @@
+export const typeOfArgumentErrorMessage = "There should be a string!"
+
+export function countNumberOfLowercase(string) {
+    let count = 0;
+    if (typeof string != "string") { throw new Error(typeOfArgumentErrorMessage) } else {
```

Comment:
in this case, it's safe (I think) but in general it is safer to use a strict comparison: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality

Reply:
done